thyrel-web/src/components/room/start/SentenceForm.tsx

```diff
@@ -6,13 +6,21 @@ import { useSessionContext } from 'hooks/SessionProvider';
 import EditIcon from '@material-ui/icons/Edit';
 import SaveIcon from '@material-ui/icons/Save';
 import { useRandomSentence } from 'hooks/useRandomSentence';
+import { useTimerEvent } from 'hooks/useTimerInterval';
 
 export default function StartForm() {
-  const { currentElement, onSave } = useSessionContext();
+  const { session, currentElement, onSave, autoSave } = useSessionContext();
   const [sentence, setSentence] = React.useState('');
   const isEditing = Boolean(!currentElement?.finishAt);
   const defaultSentence = useRandomSentence();
 
+  useTimerEvent({
+    finishAt: new Date(session?.stepFinishAt || ''),
+    timeDuration: session?.timeDuration || 60,
+    onFinish: () => autoSave(sentence),
```

Comment:
donc ici il faut directenement appelé le `client` ; 

```suggestion
    onFinish: () => {
      client<HolyElement>(`element/auto/${currentElement.id}`, {
        token: getToken(),
        method: 'PATCH',
        data: { text: sentence },
      });
    },
```